system/Config/Routes.php

```diff
@@ -20,4 +19,4 @@
  */
 
 // CLI Catchall - uses a _remap to call Commands
-$routes->cli('ci(:any)', '\CodeIgniter\CLI\CommandRunner::index/$1');
+// $routes->cli('ci(:any)', '\CodeIgniter\CLI\CommandRunner::index/$1');
```

Comment:
I think these lines are not necessary.

Reply:
Maybe then it makes sense to delete this file altogether and remove its connection from the application configuration?
https://github.com/codeigniter4/CodeIgniter4/blob/5879a3ade59dd25dc299e351e85ee51d3633cd16/app/Config/Routes.php#L10-L12